cumulusci/salesforce_api/package_install.py

```diff
@@ -48,6 +48,43 @@ class PackageInstallOptions(CCIModel):
     password: Optional[str]
     security_type: SecurityType = SecurityType.FULL
 
+    @staticmethod
+    def from_task_options(task_options: dict) -> "PackageInstallOptions":
+        options = PackageInstallOptions()  # all parameters are defaulted
+
+        try:
+            if "security_type" in task_options:
+                options.security_type = SecurityType(task_options["security_type"])
+            if "activate_remote_site_settings" in task_options:
+                options.activate_remote_site_settings = process_bool_arg(
+                    task_options["activate_remote_site_settings"]
+                )
+            if "name_conflict_resolution" in task_options:
+                options.name_conflict_resolution = NameConflictResolution(
+                    task_options["name_conflict_resolution"]
+                )
+            if "password" in task_options:
+                options.password = task_options["password"]
+        except ValueError as e:
+            raise TaskOptionsError(f"Invalid task options: {e}")
+
+        return options
```

Comment:
Maybe I'm missing something, but I think this could be done more concisely but equivalently like this:
```
task_options = {**task_options, activate_remote_site_settings=process_bool_arg(
    task_options.get("activate_remote_site_settings", True)
)
options = PackageInstallOptions(**task_options)
```

(By default, pydantic ignores extra fields in input)

Reply:
It is more concise, but I find the explicit form more readable. I also would prefer not to create a second location (in addition to the Pydantic model) where `activate_remote_site_settings` gets defaulted.